system_test/common/helpers/aest_docker.erl

```diff
@@ -223,19 +223,46 @@ setup_node(Spec, BackendState) ->
         end,
     HardForkVars =
         case maps:find(hard_forks, Spec) of
-            error -> #{};
             {ok, HardForks} ->
-                #{hard_forks_present => [#{}],
```

Comment:
Did this key have a special meaning in the mustache library? I cannot find any use.

Reply:
I tried to look it up using grep, it's not used anywhere.

Reply:
I see it was used in past test. Ok.